bin/add-cmderplugin.ps1

```diff
@@ -0,0 +1,161 @@
+<#
+.Synopsis
+    Add a plugin to Cmder
+.DESCRIPTION
+    Use this script to add a plugin the currently running Cmder
+
+    This script downloads and unpacks sources, configures users aliases, and runs pre and post commands defined in the supplied cmder-plugin-[name].json file.
+
+    You will need to make this script executable by setting your Powershell Execution Policy to Remote signed.
+
+    Then unblock the script for execution with UnblockFile .\add-cmderplugin.ps1
+.EXAMPLE
+    .\add-cmderplugin.ps1 -PluginPath '~/custom/cmder-plugin-[name].json'
+
+    Reads the source json data and excutes the steps to install nad configue the plugin.
+.EXAMPLE
+    .\add-cmderplugin.ps1 -PluginPath '~/custom/cmder-plugin-[name].json' -verbose
+
+    Reads the source json data and excutes the steps to install nad configue the plugin and see more detail as to what is going on.
+.NOTES
+    AUTHORS
+    Dax Games, Samuel Vasko, Jack Bennett
+    Part of the Cmder project.
+.LINK
+    http://cmder.net/ - Project Home
+#>
+
+[CmdletBinding(SupportsShouldProcess=$true)]
+Param(
+    # CmdletBinding will give us;
+    # -verbose switch to turn on logging and
+    # -whatif switch to not actually make changes
+
+    # Path to the vendor configuration source file
+    [string]$PluginPath = "$env:CMDER_ROOT\vendor\sources-plugins.json",
```

Comment:
I don't understand your comment.  'add-cmder-plugin.ps1' should be a part of the core cmder.  It could be updated separately if necessary by simply copying the file and utils.ps1 from the source repo into an installed binary release.